internal/services/monitor/monitor_aad_diagnostic_setting_resource_test.go

```diff
@@ -49,12 +53,45 @@ func TestAccMonitorAADDiagnosticSetting(t *testing.T) {
 	}
 }
 
+func deleteExistingAcctestAADDiagnosticSetting(t *testing.T) func() {
```

Comment:
This is intended to remove the current test resources from the tenant, as they are not under a resource group and may not be automatically cleaned up by certain tools.

<img width="1382" height="665" alt="Image" src="https://github.com/user-attachments/assets/23c3996c-77a3-4d87-b818-2a9eba0b5653" />